app/src/main/java/ro/code4/deurgenta/ui/register/RegisterActivity.kt

```diff
@@ -0,0 +1,41 @@
+package ro.code4.deurgenta.ui.register
+
+import android.os.Bundle
+import android.text.Html
+import android.text.method.LinkMovementMethod
+import kotlinx.android.synthetic.main.activity_login.*
+import kotlinx.android.synthetic.main.activity_register.*
+import org.koin.android.viewmodel.ext.android.viewModel
+import ro.code4.deurgenta.R
+import ro.code4.deurgenta.ui.base.BaseAnalyticsActivity
+
+class RegisterActivity : BaseAnalyticsActivity<RegisterViewModel>() {
+
+    override val layout: Int
+        get() = R.layout.activity_register
+    override val screenName: Int
+        get() = R.string.analytics_title_register
+    override val viewModel: RegisterViewModel by viewModel()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+
+        clickListenersSetup()
+    }
+
+    private fun clickListenersSetup() {
+        termsAndConditionsSetup()
+    }
+
+    private fun termsAndConditionsSetup() {
+        var termsLink = "https://www.google.com"
```

Comment:
could you please extract this link in the properties files?

Reply:
or even better, extract the complete text with link in the strings.xml file : https://stackoverflow.com/questions/13425002/android-html-in-strings-xml